doc/manual/build/man/dnst-key2ds.1

```diff
@@ -0,0 +1,80 @@
+.\" Man page generated from reStructuredText.
```

Comment:
Do we need to check in generated man pages?

Reply:
As it currently stands yes.